README.md

```diff
@@ -29,83 +29,72 @@ dart pub global activate very_good_cli
+  flutter_plugin    Creates a new very good federated Flutter plugin in the specified directory.
+
+Run "very_good help" to see global options.
 ```
 
 #### Usage
 
 ```sh
 # Create a new Flutter app named my_app
-very_good create my_app --desc "My new Flutter app"
+very_good create flutter_app my_app
 
 # Create a new Flutter app named my_app with a custom org
-very_good create my_app --desc "My new Flutter app" --org "com.custom.org"
+very_good create flutter_app my_app --desc "My new Flutter app" --org "com.custom.org"
 
 # Create a new Flutter app named my_app with a custom application id
-very_good create my_app --desc "My new Flutter app" --application-id "com.custom.app.id"
+very_good create flutter_app my_app --desc "My new Flutter app" --application-id "com.custom.app.id"
 
 # Create a new Flame game named my_game
-very_good create my_game -t flame_game --desc "My new Flame game"
+very_good create flame_game my_game --desc "My new Flame game"
 
 # Create a new Flutter package named my_flutter_package
-very_good create my_flutter_package -t flutter_pkg --desc "My new Flutter package"
+very_good create flutter_package my_flutter_package --desc "My new Flutter package"
 
 # Create a new Dart package named my_dart_package
-very_good create my_dart_package -t dart_pkg --desc "My new Dart package"
+very_good create dart_package my_dart_package --desc "My new Dart package"
+
+# Create a new Dart package named my_dart_package that is publishable
+very_good create dart_package my_dart_package --desc "My new Dart package" --publishable
 
 # Create a new Dart CLI application named my_dart_cli
-very_good create my_dart_cli -t dart_cli --desc "My new Dart CLI package"
+very_good create dart_cli my_dart_cli --desc "My new Dart CLI package"
 
 # Create a new Dart CLI application named my_dart_cli with a custom executable name
-very_good create my_dart_cli -t dart_cli --desc "My new Dart CLI package" --executable-name my_executable_name
+very_good create dart_cli my_dart_cli --desc "My new Dart CLI package" --executable-name my_executable_name
 
 # Create a new Flutter plugin named my_flutter_plugin (all platforms enabled)
-very_good create my_flutter_plugin -t flutter_plugin --desc "My new Flutter plugin"
+very_good create flutter_plugin my_flutter_plugin --desc "My new Flutter plugin"
 
-# Create a new Flutter plugin named my_flutter_plugin (some platforms disabled)
-very_good create my_flutter_plugin -t flutter_plugin --desc "My new Flutter plugin" --windows false --macos false --linux false
+# Create a new Flutter plugin named my_flutter_plugin (some platforms only)
+very_good create flutter_plugin my_flutter_plugin --desc "My new Flutter plugin" --platforms android,ios,macos
 
 # Create a new docs site named my_docs_site
-very_good create my_docs_site -t docs_site
+very_good create docs_site my_docs_site
 ```
 
+#### Migrating to 0.10.0
+
+⚠️ The version 0.10.0 changed the syntax of `very_good create`. Learn how to migrate [here][migrating_link].
```

Comment:
```suggestion
⚠️ The version 0.10.0 changed the syntax of `very_good create`. Learn how to migrate [here][migrating_link].
```
my question is: is it a migration or just a syntax update?

Reply:
I really don't know the difference between those two

Reply:
```suggestion
⚠️ The version 0.10.0 changed the syntax of the `very_good create` command. Learn about the changes [here][new_syntax_link].
```